package.json

```diff
@@ -55,4 +55,4 @@
     "enzyme-to-json": "^3.6.1",
     "react-test-renderer": "^16.14.0"
   }
-}
+}
```

Comment:
Did you mean to change anything in this file? We may need to just revert this file if not.